src/components/movement-form/index.tsx

```diff
@@ -289,6 +294,8 @@ export default function MovementForm({
 
           <Input
             label="Atribuição"
+            readOnly
+            defaultValue={user?.job}
```

Comment:
Seria bom deixar os campos como desabilitados.
```suggestion
            defaultValue={user?.job}
            disabled
```